content/en/tools/starter-kits.md

```diff
@@ -29,6 +29,7 @@ The following starter kits are developed by active members of the Hugo community
 * [Blaupause][]. Blaupause is a developer-friendly Hugo starter kit based on Gulp tasks. It comes ES6-ready with several helpers for SVG and fonts and basic structure for HTML, SCSS, and JavaScript.
 * [hugulp][]. hugulp is a tool to optimize the assets of a Hugo website. The main idea is to recreate the famous Ruby on Rails Asset Pipeline, which minifies, concatenates and fingerprints the assets used in your website.
 * [Atlas][]. Atlas is a Hugo boilerplate designed to speed up development with support for Netlify, Hugo Pipes, SCSS & more. It's actively maintained and contributions are always welcome.
+* [JAMStack Themes][]. JAMStack themes is a collection of site themes filterable by static site generator and supported CMS to help build CMS-connected sites using Hugo (linking to Hugo-specific themes).
```

Comment:
I would not put it in Starter Kits, as it's more or less a collection of themes. Better fits in "Other Community Projects"? https://gohugo.io/tools/other/ @bep

Reply:
Ok. I'll submit a pull request for that instead